components/Pill.tsx

```diff
@@ -45,6 +48,7 @@ const BasePill = styled.button<{
 	$fullWidth?: boolean;
 	$roundedCorner?: boolean;
 	$weight?: FontWeight;
+	$background?: PillColor;
```

Comment:
I think this background prop is confusing with the cross over of color prop. Really we should just have some color variants that derive style from `newTheme.pill[variant]`. Maybe we just need to add a new pill style there.

Reply:
Good idea. Updated.